invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/controls/DeleteButton/DeleteButton.js

```diff
@@ -119,7 +119,7 @@ export class DeleteButtonComponent extends Component {
               {i18next.t("Cancel")}
             </Button>
             <Button
-              {...color}
+              className={color}
```

Comment:
Thank you! Can you check if the `warning` class works? It looks like that it does not exist in ReactSUI: https://react.semantic-ui.com/elements/button/